.vscode/launch.json

```diff
@@ -1,35 +1,35 @@
 {
-    "version": "0.2.0",
-    "configurations": [
-        {
-            "type": "node",
-            "request": "launch",
-            "name": "Main - Windows",
-            "args": [
-                "${workspaceFolder}\\dist\\src\\Skyra.js"
-            ],
-            "protocol": "inspector",
-            "internalConsoleOptions": "openOnSessionStart",
-            "env": {
-                "NODE_ENV": "development"
-            },
-            "console": "internalConsole",
-            "outputCapture": "std"
-        },
-        {
-            "type": "node",
-            "request": "launch",
-            "name": "Main - Linux",
-            "args": [
-                "${workspaceFolder}/dist/src/Skyra.js"
-            ],
-            "protocol": "inspector",
-            "internalConsoleOptions": "openOnSessionStart",
-            "env": {
-                "NODE_ENV": "development"
-            },
-            "console": "internalConsole",
-            "outputCapture": "std"
-        }
-    ]
-}
+	"version": "0.2.0",
+	"configurations": [
+		{
+			"type": "node",
+			"request": "launch",
+			"name": "Main - Windows",
+			"args": [
+				"${workspaceFolder}\\dist\\src\\Skyra.js"
```

Comment:
Same thing here

Reply:
```suggestion
				"${workspaceFolder}\\dist\\Skyra.js"
```

Reply:
gdi I searched for `src/Skyra`, forgetting about the `src\\Skyra` lol. Thanks for catching it guys.